README.adoc

```diff
@@ -533,6 +533,10 @@ relabel "request_uri" {
 }
 ----
 
+### JSON log_format
+
+You can use JsonParser by setting the parser argument of the command line or config-file to json.
```

Comment:
:+1: for also thinking of the documentation. Just a few suggestions regarding the wording.

```suggestion
You can use the JSON parser by setting the `--parser` command line flag or `parser` config file property to `json`.
```

Reply:
Sorry, English is not my native language. Thanks very much for your correction.👍